server/settings/tools.py

```diff
@@ -548,10 +559,13 @@ class GlobalToolsModel(BaseSettingsModel):
             },
             {
                 "product_types": [
-                    "simpleUnrealTexture"
+                    "simpleUnrealTexture",
+                    "image",
+                    "textures"
```

Comment:
This change would now also force publishes from `standalonepublisher` that are just `image` and `textures` publishes to be renamed to the new template - that sounds incorrect behavior and overly specifically targeted to ingesting for Unreal.